harvest/utils.py

```diff
@@ -1,20 +1,78 @@
 import re
 import random
 import datetime as dt
+from enum import IntEnum, auto
 
 import pytz
 import pandas as pd
 
-
-def expand_interval(interval: str):
-    time_search = re.search("([0-9]+)(MIN|HR|DAY)", interval)
-    value = int(time_search.group(1))
-    unit = time_search.group(2)
-    return value, unit
-
-
-def interval_to_timedelta(interval: str) -> dt.timedelta:
-    expanded_units = {"DAY": "days", "HR": "hours", "MIN": "minutes"}
+class Interval(IntEnum):
+    SEC_15 = auto()
+    MIN_1 = auto()
+    MIN_5 = auto()
+    MIN_15 = auto()
+    MIN_30 = auto()
+    HR_1 = auto()
+    DAY_1 = auto()
```

Comment:
Brokers like alpaca and polygon can taken in any interval, should we limit the intervals to this?

Reply:
Yeah these are the standard intervals guaranteed to work on all brokers